bokehjs/src/lib/models/widgets/autocomplete_input.ts

```diff
@@ -43,6 +43,7 @@ export class AutocompleteInputView extends TextInputView {
       this.input_el.focus()
       this._hide_menu()
     }
+    super.change_input()
```

Comment:
@ltalirz I don't see how this alone an be sufficient, unless I am mistaken the `strict` property is not being used or referenced anywhere now, which means it can't have any effect on behavior (which means one of strict or non-strict mode must be broken)

Reply:
Oops, this should only be done if strict is false. Will fix.

Reply:
This actually gets to another issue: most widgets now have good integration tests, but it's also definitely still work in progress and it seems `Autocomplete` did not have any full callback integration tests. If you are up for adding corresponding tests for "non-strict" mode alongside the new tests for strict mode that would certainly be valuable and appreciated (but no worries if its outside your bandwidth budget for this)

Reply:
It's no problem to add some more tests for non-strict mode - if you have anything specific in mind, just let me know.
If possible, I'd prefer to do this in a separate PR since we need this functionality and we'll still need to propagate the changes through panel